apps/parsley/src/hooks/useSections/index.ts

```diff
@@ -108,32 +114,39 @@ export const useSections = ({
     [],
   );
 
+  /**
+   * openSectionsContainingLineNumbers Will update the current section state. If the next state is the
+   * same as the current state the function will return true and false otherwise.
+   * @param param0 is an object with a lineNumber key(s).
+   * @param param0.lineNumbers is a number or an array of numbers that represent raw log line numbers.
+   * @returns true if the sectionState was updated and false otherwise
+   */
+  const openSectionsContainingLineNumbers = ({
+    lineNumbers,
+  }: {
+    lineNumbers: number[];
+  }): boolean => {
+    if (!sectionData || !sectionState) {
+      return false;
+    }
+    const [hasDiff, nextState] =
+      getSectionStateWithOpenSectionBasedOnLineNumber({
```

Comment:
`lineNumbers` maybe!
```suggestion
      getSectionStateWithOpenSectionBasedOnLineNumbers({
```
another good name for this function could be `getOpenSectionStateBasedOnLineNumbers`